examples/quickstart/inference_config.yaml

```diff
@@ -0,0 +1,26 @@
+name: inference_pipeline
```

Comment:
for other examples you mentioned in the README the option to run with `-c config`, why not for quickstart?